classes/Http/Controller/Admin/SpeakersController.php

```diff
@@ -23,30 +23,55 @@
 use OpenCFP\Domain\Speaker\SpeakerProfile;
 use OpenCFP\Http\Controller\BaseController;
 use Symfony\Component\HttpFoundation\Request;
-use Symfony\Component\HttpFoundation\Session;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Twig_Environment;
 
 class SpeakersController extends BaseController
 {
+    private $authentication;
+    private $accounts;
+    private $airports;
+    private $capsule;
+    private $applicationArrival;
+    private $applicationDeparture;
+    private $applicationAirport;
```

Comment:
Can we add docblocks here?

Reply:
Sure, I'll add doc blocks to all newly introduced fields.

Reply:
To explain myself: I usually leave them out for dependencies, because the type-hints on the constructor already document the types of the fields.